packages/phone-number-privacy/combiner/src/pnp/endpoints/quota/action.ts

```diff
@@ -0,0 +1,34 @@
+import { ErrorMessage, PnPQuotaRequest } from '@celo/phone-number-privacy-common'
+import { CombineAction } from '../../../common/combine'
+import { IO } from '../../../common/io'
+import { Session } from '../../../common/session'
+import { OdisConfig } from '../../../config'
+import { CombinerThresholdStateService } from '../../services/thresholdState'
+
+export class PnPQuotaAction extends CombineAction<PnPQuotaRequest> {
```

Comment:
nit: `PnP` -> `Pnp` in the newly added classes/types/etc. to be consistent with existing naming